package.json

```diff
@@ -1,39 +1,78 @@
 {
   "name": "@nextcloud/files",
-  "version": "2.1.0",
-  "description": "",
+  "version": "3.0.0",
+  "description": "Nextcloud files utils",
   "main": "dist/index.js",
+  "module": "dist/index.esm.js",
   "types": "dist/index.d.ts",
-  "scripts": {
-    "build": "babel ./lib --out-dir dist --extensions '.ts,.tsx' --source-maps && tsc --emitDeclarationOnly",
-    "build:doc": "typedoc --out dist/doc lib && touch dist/doc/.nojekyll",
-    "check-types": "tsc",
-    "dev": "babel ./lib --out-dir dist --extensions '.ts,.tsx' --watch",
-    "test": "jest",
-    "test:watch": "jest --watchAll"
+  "exports": {
+    ".": {
+      "import": "./dist/index.esm.js",
+      "require": "./dist/index.js"
+    }
   },
+  "files": [
+    "dist"
+  ],
+  "author": "Christoph Wurst <christoph@winzerhof-wurst.at>",
+  "contributors": [
+    "John Molakvoæ <skjnldsv@protonmail.com>"
+  ],
+  "license": "AGPL-3.0-or-later",
   "keywords": [
-    "nextcloud"
+    "nextcloud",
+    "files",
+    "library"
   ],
-  "author": "Christoph Wurst",
-  "license": "GPL-3.0-or-later",
-  "dependencies": {
-    "@nextcloud/l10n": "^1.3.0",
-    "core-js": "^3.6.4"
+  "scripts": {
+    "build": "rollup --config rollup.config.js",
+    "dev": "echo 'No dev build available, production only' && npm run build",
+    "watch": "rollup --config rollup.config.js --watch",
+    "lint": "eslint --ext .js,.vue lib",
+    "lint:fix": "eslint --ext .js,.vue lib --fix",
+    "test": "jest",
+    "test:watch": "jest --watchAll --verbose true",
+    "test:coverage": "jest --coverage"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/nextcloud/nextcloud-files.git"
+  },
+  "bugs": {
+    "url": "https://github.com/nextcloud/nextcloud-files/issues"
   },
+  "engines": {
+    "node": "^14.0.0",
+    "npm": "^7.0.0"
+  },
+  "homepage": "https://github.com/nextcloud/nextcloud-files",
   "devDependencies": {
-    "@babel/cli": "^7.8.4",
-    "@babel/core": "^7.8.7",
-    "@babel/preset-env": "^7.8.7",
-    "@babel/preset-typescript": "^7.8.3",
-    "@nextcloud/browserslist-config": "^2.0.0",
-    "babel-jest": "^27.0.2",
-    "babel-plugin-transform-class-properties": "^6.24.1",
-    "jest": "^27.0.4",
-    "typedoc": "^0.22.4",
-    "typescript": "^4.0"
+    "@nextcloud/babel-config": "^1.0.0",
```

Comment:
this sneaks in a bunch of unrelated dependency upgrade

Reply:
Does it? Seems more like the proper upgrade from [the configs that are being used](https://github.com/nextcloud/nextcloud-files/blob/master/babel.config.js), no?
It's part of the rollup upgrade to properly use them